app/frontend/helpers/debugLog.js

```diff
@@ -1,7 +1,7 @@
 const CARDANOLITE_CONFIG = require('../config').CARDANOLITE_CONFIG
 
 function debugLog(message) {
-  if (CARDANOLITE_CONFIG.CARDANOLITE_ENABLE_DEBUGGING) {
+  if (CARDANOLITE_CONFIG && CARDANOLITE_CONFIG.CARDANOLITE_ENABLE_DEBUGGING) {
```

Comment:
is this useful? who would bebug without config?

Reply:
the tests fail without this. There is already an issue to refactor debug logging: https://github.com/vacuumlabs/cardanolite/issues/161

Reply:
but it fails only if something is logged which isn't if everything goes as expected. Ok, dropping this change for now.